src/flypad-backend/src/FlyPadBackend.cpp

```diff
@@ -64,12 +68,45 @@ bool FlyPadBackend::initialize() {
   result &= SimConnect_MapClientEventToSimEvent(hSimConnect, Events::KEY_TUG_HEADING_EVENT, "KEY_TUG_HEADING");
   result &= SimConnect_MapClientEventToSimEvent(hSimConnect, Events::KEY_TUG_SPEED_EVENT, "KEY_TUG_SPEED");
 
-  // initialize submodules
-  lightPresetPtr->initialize();
-  aircraftPresetPtr->initialize();
-  pushbackPtr->initialize();
+  if (result != S_OK) {
+    std::cout << "FLYPAD_BACKEND: Events definition failed! " << std::endl;
+  }
+
+  result &= SimConnect_MapClientDataNameToID(hSimConnect, "IVAO Altitude Data", ClientData::IVAO);
+  //result &= SimConnect_CreateClientData(hSimConnect, ClientData::IVAO, sizeof(ThirdPartyDataIVAO), SIMCONNECT_CREATE_CLIENT_DATA_FLAG_DEFAULT);
+  result &= SimConnect_AddToClientDataDefinition(hSimConnect, DataStructureIDs::SelcalIVAODataID, 0, SIMCONNECT_CLIENTDATATYPE_INT8);
+  result &= SimConnect_AddToClientDataDefinition(hSimConnect, DataStructureIDs::VolumeCOM1DataID, 1, SIMCONNECT_CLIENTDATATYPE_INT8);
+  result &= SimConnect_AddToClientDataDefinition(hSimConnect, DataStructureIDs::VolumeCOM2DataID, 2, SIMCONNECT_CLIENTDATATYPE_INT8);
+  result &= SimConnect_AddToClientDataDefinition(hSimConnect, DataStructureIDs::AllIVAODataID, 0, sizeof(ThirdPartyDataIVAO));
```

Comment:
Try to group these IDs if possible - similar to PushbackDataID

Reply:
I've done it this way because it was done this way in Altitude (IVAO). I agree it would make things simpler with one ID only. I'll give it a try tonight

Reply:
Done